main.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/kubernetes-sigs/container-object-storage-interface-csi-adapter/cmd"
+	"github.com/golang/glog"
+)
+
+func main() {
+	sigs := make(chan os.Signal, 1)
+	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM, syscall.SIGSEGV)
```

Comment:
Not sure we want to 'properly handle' `SIGSEGV`. IMO a segmentation fault should crash an application unless the app can really do something 'intelligent' about it.

Reply:
SEGFault should not lead to forced termination of any I/O in progress. The whole point of gracefully handling this signal to complete I/O in progress before exiting. This should still be here and added to the other repos as well if it’s not already there

Reply:
That makes sense to make, I have removed that signal for the time being